### ISSUE TITLE
[HTTPS] Transfert ne fonctionne plus

### ISSUE THREAD

**Issue by WITNicolas** [witsa/synapps#696]
On ne peut plus transférer les SYNAPP via HTTPS

REDY 14.5.8
STUDIO 1.2.0

**Comment by WIToliv** [comment]
Bonjour,
Le problème persiste, j'arrive pas à publier ma synapps sur un redy distant en https.
REDY 14.5.2
Studio 1.2.1

**Comment by Ratrasse** [comment]
Bonjour,
Nous avons constaté que en fonction de la taille du fichier a tranferer vers le REDY un error de time out nous arribe dans le cas d'ussage du HTTPS
REDY 14.5.2
Studio 1.2.1
Important de resoudre, car l'ussage HTTPS c'est imprescindible sur certains sites.

**Comment by manufitoussiwit** [comment]
Oui, il semble qu'il ait un soucis en générale avec les REDY en HTTPS et les fichiers plus ou moins gros. ça n'impacte pas que Synapps et Studio mais même l'interface du REDY. L'équipe REDY travail dessus.

**Comment by manufitoussiwit** [comment]
Pour ceux qui ont eu ce soucis, l'avez vous toujours avec des REDY en version 16 ?

**Comment by WITNicolas** [comment]
Pas de soucis de mon coté, je viens de faire à l'instant. 16.1.2

**Comment by Raphwit** [comment]
Corrigé version 16+ du REDY
